src/Autosuggest.js

```diff
@@ -140,6 +145,10 @@ export default class Autosuggest extends Component {
     }
 
     this.setState(newState);
+
+    this.props.onSuggestionUnfocused(lastFocusedSuggestion);
```

Comment:
I believe `onSuggestionUnfocused` shouldn't be called if `lastFocusedSuggestion === null`. What do you think?